Port/Port/Models/Ports.cs

```diff
@@ -16,13 +16,13 @@
 
 namespace Port.Models
 {
-    public class Port
+    public class Ports
```

Comment:
Promijeni u Port. Svaki model opisuje jedniku, a onda u repozitoriju spremaš luke. Ports navodi čitatelja da u modelu spremaš podatke o više luka, koliko god to glupo zvučalo, ne smiješ ostaviti sumnje u svom kodu.

Reply:
Vidim sada da u modelu opisuješ nekakvo putovanje. Luka u Splitu nema departure time, ali brod ima. Molim te promijeni ime modela u Ship da bolje preneseš čitatelju ono što radiš. To podrazumijeva da ćeš promijeniti i ime controllera i repozitorija da nije više PortController i PortRespoitory nego ShipRepository i ShipController.